src/components/SpreadsheetUploader/index.tsx

```diff
@@ -13,14 +13,26 @@ const acceptedTypes = [
   'text/csv',
 ];
 
-const displayCell = (cellData) => {
+const displayCell = (cellData: any) => {
   if (cellData instanceof Date) {
     return cellData.toLocaleString();
   } else {
     return cellData;
   }
 };
 
+type Props = {
+  open: boolean;
+  setOpen: ({}) => void;
```

Comment:
Again, this is an inaccurate type for this callback function. Please correctly type the parameter.